tests/src/fixtures/permissions/integrated/user.role.content_author.yml

```diff
@@ -39,16 +39,6 @@ permissions:
   - 'access components'
   - 'access content elements group'
   - 'access content overview'
-  - 'access cpt_cat_accordion_components cohesion_component_category group'
```

Comment:
did we added new test to verify these permissions exist? this is for all SS permission removed in this PR.